Meadowlark-js/packages/meadowlark-core/src/model/DocumentReference.ts

```diff
@@ -4,18 +4,21 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 import { PropertyType, ModelType } from '@edfi/metaed-core';
+import { DocumentIdentity } from './DocumentIdentity';
 
-export type ReferentialConstraint = {
+export type DocumentReference = {
```

Comment:
Document reference is an identity plus information on the document/entity type and optionally super/subclass info. Some MetaEd leakiness is still here for now.